tmt/cli.py

```diff
@@ -18,6 +18,9 @@
 import tmt.templates
 import tmt.options
 
+# Version string is replaced before building the package
+VERSION = 'running from the source'
```

Comment:
Could this be made as `tmt.__version__` instead? Check https://stackoverflow.com/questions/20180543/how-to-check-version-of-python-modules leading to https://www.python.org/dev/peps/pep-0396/ and https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
Good idea, @lukaszachy. Moved the version to `tmt.__version__`.